tools/its.py

```diff
@@ -582,17 +582,19 @@ def termin_suchen(self, plz: str, zeitrahmen: dict):
             res_json = res.json()
             terminpaare = res_json.get("termine")
             if terminpaare:
-                terminpaare_angenommen = {
-                    tp for tp in terminpaare
-                    if terminpaar_im_zeitrahmen(tp, zeitrahmen)
-                }
-                for tp_abgelehnt in set(terminpaare) - terminpaare_angenommen:
-                    self.log.info(
-                        "Termin gefunden - jedoch nicht im entsprechenden Zeitraum")
-                    for num, termin in enumerate(tp_abgelehnt, 1):
-                        ts = datetime.fromtimestamp(termin["begin"] / 1000).strftime(
-                            '%d.%m.%Y um %H:%M Uhr')
-                        self.log.info(f"{num}. Termin: {ts}")
+                terminpaare_angenommen = list()
+                for tp in terminpaare:
+                    if terminpaar_im_zeitrahmen(tp, zeitrahmen):
+                        terminpaare_angenommen.append(tp)
+
+                for tp_abgelehnt in terminpaare:
+                    if terminpaare in terminpaare_angenommen:
+                        self.log.info(
+                            "Termin gefunden - jedoch nicht im entsprechenden Zeitraum")
+                        for num, termin in enumerate(tp_abgelehnt, 1):
+                            ts = datetime.fromtimestamp(termin["begin"] / 1000).strftime(
+                                '%d.%m.%Y um %H:%M Uhr')
+                            self.log.info(f"{num}. Termin: {ts}")
```

Comment:
Was war hier das Problem, weshalb du es geändert hast?

Reply:
`   for tp_abgelehnt in set(terminpaare) - terminpaare_angenommen:`

Hatte einen Fehler verursacht

Reply:
Und welchen Fehler? Hab das in der `python3`-Konsole getestet und gefolgert, dass es funktioniert.

Reply:
Soweit ichs noch richtig weiß wars: unhashable type 'list'

Reply:
Ah, verstehe.